### ISSUE TITLE
Pls there are plans to put out servers that are not based on port 80,443 because my server has both ports disabled.

### ISSUE THREAD

**Issue by wwfkmd1** [netbirdio/netbird#3123]
Since my server can't use port 80,443, I want to use a different port instead, is there a similar plan?

**Comment by Gauss23** [comment]
I don't see why it wouldn't be possible to not use port 80 and 443. Without port 80 you'll loose Let's encrypt and you would need to add your own certificate for the management server or you'll need to switch to the DNS challenge for LE.

Where do you get servers with ports 80 and 443 disabled? This is a strange approach.

**Comment by nmincone** [comment]
I agree with this initiative, it would be great to be able to use alternative ports. I am running into the same issue as I have NB running in a Proxmox Debian VM asking for ports 80/443 on a different IP then my Debian Docker server VM which is already using those ports.

**Comment by PizzaLovingNerd** [comment]
Hello, this is a duplicate of #1794 and #1786.
